test/surface/run-solid-test-suite.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -i
+npm start &
+git clone https://github.com/solid/solid-crud-tests
+cd solid-crud-tests
+npm install
+# needed due to git dependency, until latest rdflib code is on npm:
```

Comment:
Should we perhaps use a specific commit of `solid-crud-tests` to avoid it breaking?

Reply:
Yes, I should publish it to npm probably

Reply:
I now used `git checkout v0.1.1`